content/en/docs/creating-addons/top-level-operator/addon-package.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Addon Package
+linkTitle: Addon Package
```

Comment:
```suggestion
title:  Plug and Play Addon
linkTitle: Plug and Play Addon
```
We are referring to this everywhere as "Addons Plug and Play" or "PnP", for consistency we should here as well.